discovery-provider/src/api/v1/utils/extend_notification.py

```diff
@@ -388,6 +390,42 @@ def extend_trending(action: NotificationAction):
     return notification
 
 
+def extend_trending_playlist(action: NotificationAction):
```

Comment:
this notif hasnt been implemented right? guessing you're just getting it out of the way while youre here?

Reply:
on client side? i was thinking since it's just a new "type" in database it doesn't technically need changes?

Reply:
oh yeah great call, i can def remove this one for now :)

Reply:
nah no worries i was just double checking, up to you how you wanna roll it out!